articles/deciding-between-cloud-spanner-and-microsoft-cosmos-db/index.md

```diff
@@ -0,0 +1,55 @@
+Creating a globally distributed database would traditionally require a lot of time and effort. You had to host the database in an own data center and use your own connections. Today&#39;s advancement in cloud computing and platforms as a service (PaaS) has made it easier to create globally distributed and scalable databases. Particularly, the use of SQL-based databases has become more popular in the recent past. These databases provide you with a customized experience of sorting and processing large amounts of raw data. Many database providers exist, and they work on different nodes of the SQL database engines, including [Microsoft Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction) and [Google Cloud Spanner](https://cloud.google.com/spanner).
+
+This article will discuss the features of both Cloud Spanner and Microsoft Cosmos DB and highlight their advantages and disadvantages. It will also compare the two to help you better choose the cloud database to use in the future.
+
+### Features of Google Cloud Spanner
+
+Google Spanner is a fully managed, scalable, relational database management service.
+
+The platform can scale millions of nodes across multiple regions. Besides being integrated with access and identity management, it provides features such as [logging and auditing](https://cloud.google.com/spanner/docs/audit-logging).
+
+Google Cloud Spanner&#39;s instance is comparable to a relational database management system ([RDBMS](https://www.codecademy.com/articles/what-is-rdbms-sql)) server. This instance is composed of one or multiple databases, which use the same [compute](https://www.zdnet.com/article/what-is-cloud-computing-everything-you-need-to-know-about-the-cloud/) and [storage](https://searchstorage.techtarget.com/definition/cloud-storage) resources. These resources are allocated during [instance](https://cloud.google.com/spanner/docs/instances) creation. Two main configurations, [Regional and Multi-Regional](https://cloud.google.com/spanner/docs/instances), control the compute and storage resources.
+
+Cloud Spanner supports automatic data replication. Instance configuration (the first configuration) determines the number of copies ([replicas](https://cloud.google.com/spanner/docs/replication)) to be created and their placement. Regional configuration allows data replication across three zones. These zones must come from within a single selected region. Multi-Regional configuration, on the other hand, supports data replication across four zones. These zones can come from different regions depending on what continent the user specifies.
```

Comment:
Include a map of GCP zones and regions. It's not clear what these conditions are, and how they are useful. A picture should solve that problem.